src/modules/new-request-form/ticket-form-field/TicketFormField.tsx

```diff
@@ -30,10 +34,21 @@ export function TicketFormField({
         newUrl.searchParams.append(key, value);
       }
 
+      sessionStorage.setItem(key, crypto.randomUUID());
```

Comment:
We don't seem to be using the value. Why setting it as `crypto.randomUUID()`?

I'm wondering if it would be helpful to make it both the key and the value more descriptive i.e. `setFocusToTicketFormField: true`? What do you think?

Reply:
Pushed a change aligning it with your feedback. Please have another look when you have time 🙏🏼